src/main/clojure/aerospike_clj/collections.clj

```diff
@@ -0,0 +1,19 @@
+(ns aerospike-clj.collections
+  (:import (java.util ArrayList Collection Collections List)
+           (java.util.function Consumer)))
+
+(defn ->map
```

Comment:
it's just an alternative `mapv` implementation why not just call the fn `map`?

Reply:
Mainly because it shadows the `clojure.core/map` function, and I get this warning:
```
WARNING: map already refers to: #'clojure.core/map in namespace: aerospike-clj.collections, being replaced by: #'aerospike-clj.collections/map
```

Reply:
What do you think about `->list` or `listv`?